packages/dappkit/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import { CeloTokenType, GoldToken, StableToken } from '@celo/contractkit'
+import { AccountAuthRequest, serializeDappKitRequestDeeplink, SignTxRequest } from '@celo/utils'
+import { Linking } from 'expo'
+import Web3 from 'web3'
+import { TransactionObject } from 'web3/eth/types'
+
+export { AccountAuthRequest, serializeDappKitRequestDeeplink, SignTxRequest } from '@celo/utils'
```

Comment:
Why re-export here?

Reply:
To expose it to consumers of the dappkit, without them having to require it from "@celo/utils"

Reply:
Cool. index.ts seems like a strange choice of place to have other consumers import from tho.

Reply:
Is that not what we did with the old contract kit?